cli/spec/kontena/cli/volumes/list_command_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'kontena/cli/volumes/list_command'
+
+describe Kontena::Cli::Volumes::ListCommand do
+  include ClientHelpers
+  include OutputHelpers
+  include RequirementsHelper
+
+  expect_to_require_current_master
+  expect_to_require_current_master_token
+
+  mock_current_master
+
+  let(:subject) { described_class.new("kontena") }
+  let(:response)  do
+    {
+      'volumes' => [
+        { 'id' => 'test-grid/testvol',
+          'name' => 'testvol',
+          'scope' => 'testscope',
+          'driver' => 'testdriver',
+          'created_at' => Time.parse('2001-01-01 12:00:00').to_s
+        },
+        { 'id' => 'test-grid/testvol2',
+          'name' => 'testvol2',
+          'scope' => 'testscope2',
+          'driver' => 'testdriver2',
+          'created_at' => Time.parse('2001-01-02 12:00:00').to_s
+        }
+      ]
+    }
+  end
+
+  it 'lists volumes' do
+    expect(client).to receive(:get).with('volumes/test-grid').and_return(response)
+    expect{subject.run([])}.to output_table [
+      ['testvol',  'testscope', 'testdriver', a_string_matching(/\d+ days ago/)],
+      ['testvol2', 'testscope2', 'testdriver2', a_string_matching(/\d+ days ago/)]
```

Comment:
Failing on travis because the command is outputting a timestamp, instead of a `... days ago`. WTF?

```
  1) Kontena::Cli::Volumes::ListCommand lists volumes
     Failure/Error:
       expect{subject.run([])}.to output_table [
         ['testvol',  'testscope', 'testdriver', a_string_matching(/\d+ days ago/)],
         ['testvol2', 'testscope2', 'testdriver2', a_string_matching(/\d+ days ago/)]
       ]
       on line 1:
        expected: ["testvol", "testscope", "testdriver", #<RSpec::Matchers::AliasedMatcher:0x00000003624758 @description_block=#<Proc:0x00000002410c38@/home/travis/build/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/rspec-expectations-3.6.0/lib/rspec/matchers/dsl.rb:31 (lambda)>, @base_matcher=#<RSpec::Matchers::BuiltIn::Match:0x00000003624780 @expected=/\d+ days ago/, @expected_captures=nil, @matcher_name=:a_string_matching>>]
        received: ["testvol", "testscope", "testdriver", "2001-01-01 12:00:00 +0000"]
       on line 2:
        expected: ["testvol2", "testscope2", "testdriver2", #<RSpec::Matchers::AliasedMatcher:0x000000036245f0 @description_block=#<Proc:0x00000002410c38@/home/travis/build/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/rspec-expectations-3.6.0/lib/rspec/matchers/dsl.rb:31 (lambda)>, @base_matcher=#<RSpec::Matchers::BuiltIn::Match:0x00000003624618 @expected=/\d+ days ago/, @expected_captures=nil, @matcher_name=:a_string_matching>>]
        received: ["testvol2", "testscope2", "testdriver2", "2001-01-02 12:00:00 +0000"]
     # ./spec/kontena/cli/volumes/list_command_spec.rb:35:in `block (2 levels) in <top (required)>'
     # ./spec/spec_helper.rb:50:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:48:in `catch'
     # ./spec/spec_helper.rb:48:in `block (2 levels) in <top (required)>'
```

Reply:
Oh right... the `kontena volume list` command changes the output format based on stdout:

```ruby
option ['--[no-]long', '-l'], :flag, "Show full dates", default: !$stdout.tty?
```